src/main/scala/au/com/agiledigital/toolform/command/inspect/InspectCommand.scala

```diff
@@ -0,0 +1,44 @@
+package au.com.agiledigital.toolform.command.inspect
+
+import java.nio.file.Path
+
+import au.com.agiledigital.toolform.app.ToolFormError
+import au.com.agiledigital.toolform.model.Project
+import au.com.agiledigital.toolform.plugin.ToolFormCommandPlugin
+import au.com.agiledigital.toolform.reader.ProjectReader
+import com.monovore.decline._
+
+/**
+  * Prints a summary of a project definition.
+  */
+final class InspectCommand extends ToolFormCommandPlugin {
+
+  override val command: Opts[Either[ToolFormError, String]] =
+    Opts
+      .subcommand("inspect", "Inspect and print the content of the project file") {
+        Opts.option[Path]("input", short = "i", metavar = "file", help = "Input file")
```

Comment:
Is this the same as the input option for generate? If so, should it be extracted to a common location?

Reply:
Yes, it is the same type of input, a project config file.
Agree it could be lovely to extract to enforce consistency and remove duplication, but not sure how to set up the type system for abstract commands without another example or two.

Reply:
Sure.

Reply:
Doesnt the DockerComposeV3 and the new Kubernetes generators need the parsed input?

Reply:
@timechanter I think so - those other commands are still landing in the old parser format to be sure.  One thing that differs with them is output format, so it would be nice to see the variations before trying to model an abstraction here.
I can take a shot at it in a new PR asap and we can always change it again later - not a big investment either way.